gef-ui/templates/includes/exporters-address/seperate-correspondence.njk

```diff
@@ -82,4 +82,4 @@
     </form>
   </div>
 </div>
-<script src="/assets/js/correspondenceAddress.js" type="text/javascript"></script>
\ No newline at end of file
+<script src="/assets/js/correspondenceAddress.js" type="text/javascript" integrity="sha512-bhqdWe9Ie4k2bAdaRejzaWJkbviETG4zN6jeAepKRxNrC9jmgtRe7TgibTkV87G8RAMPzMVnX6MgDBGcOvgRLA==" crossorigin="anonymous"></script>
```

Comment:
can this be passed as a variable? or is there any risk of having it in viewable text form like this?